.travis.scripts/before_script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+CONFIGURE_OPTS="--enable-developer-flags --enable-coverage"
+
+if [ -n "${SSL_VERSION}" ]; then
+    CONFIGURE_OPTS="${CONFIGURE_OPTS} --with-mongodb-ssl=${SSL_VERSION}"
+fi
+
+phpize
+./configure ${CONFIGURE_OPTS}
+make all -j4
+make install
+
+# Use latest run-tests.php, as older versions don't report a non-zero exit code for failures
+wget -O run-tests.php https://raw.githubusercontent.com/php/php-src/master/run-tests.php
```

Comment:
Do we still need to do this?

Reply:
While I very much remember the problem with `run-tests.php` exiting with zero after failures, I don't quite remember what versions of PHP demonstrated the issue. There is evidence of `exit(1)` going back to [PHP-5.3](https://github.com/php/php-src/blob/PHP-5.3/run-tests.php#L952).

I did come across https://github.com/php/php-src/commit/16efc77608789234c92c1cad906417d06ec3428d, which changed the logic for `exit(1)` in 5.5. Do you recall if this was only an issue up to 5.4 (inclusive)?

If you're able to narrow it down, we can certainly get rid of this.

Reply:
I'll narrow this down in a bit and if 5.5+ reports exit codes, we can remove this.

Reply:
Confirmed that PHP 5.4 was the last version with this problem. I'll remove this.